taker-frontend/src/App.tsx

```diff
@@ -106,14 +106,26 @@ export const App = () => {
             <Disclaimer />
             <Nav walletInfo={walletInfo} connectedToMaker={connectedToMaker} />
             <Box textAlign="center" padding={3}>
-                {connectionStatus}
                 <Routes>
-                    <Route path="/wallet" element={<Wallet walletInfo={walletInfo} />} />
+                    <Route
+                        path="/wallet"
+                        element={<>
+                            <Center>
+                                <VStack>
+                                    {connectionStatus}
+                                    <Wallet walletInfo={walletInfo} />
```

Comment:
I think both of these components already have a `Center` in them somewhere. Maybe it would be better to just pass the data (`isConnected`) down and render it on each page separately?